__tests__/pet.test.js

```diff
@@ -0,0 +1,187 @@
+const Pet = require('../src/pet');
+const Parent = require('../src/pet');
+
+describe('constructor',()=>{
+    it('returns an object',()=>{
+        expect(new Pet('Fido')).toBeInstanceOf(Object);
+    })
+   
+
+    it('sets the name property',()=>{
+        const myPet = new Pet('Fido');
+        expect(myPet.name).toEqual('Fido');
+    })
+
+
+    it('has an initial age of 0',()=>{
+        const myPet = new Pet('Fido');
+        expect(myPet.age).toEqual(0);
+    })
```

Comment:
I would also add tests to check that the other conditions are initialised as expected, e.g.
```
it('has an initial hunger of 0", () => {
  ...
});

it('has an initial fitness of 10', () => {
  ...
});

it('has not children', () => {
  ...
});
```